x-pack/test_serverless/api_integration/test_suites/common/reporting/csv_v2_esql.ts

```diff
@@ -112,8 +117,8 @@ export default ({ getService }: FtrProviderContext) => {
               },
             ],
           }));
-          await reportingAPI.waitForJobToFinish(path, roleAuthc, internalReqHeader);
-          response = await supertest.get(path);
+          await reportingAPI.waitForJobToFinish(path, cookieCredentials, internalReqHeader);
+          response = await supertestWithoutAuth.get(path).set(cookieCredentials);
```

Comment:
This test was BROKEN since [the original PR](https://github.com/elastic/kibana/pull/192533/files#diff-a4e6ac4278194884b90d51ddbc82187a4a5bd451dcfb60d86fa6bc9b770067b8). The snapshot showed a 404 error. This is now fixed.

Reply:
oh no :( typical for snapshot tests in a hidden file